drawing_shape2

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+
+"""
+Fragment kodu odpowiadający za rysowanie z plikow shp metodą lineTo
+
+Problemy:
+    - jak przekazać miejsce kliknięcia myszą do innych funkcji?
+    - jak ponownie wywołać rysowanie skoro już zostało zakończone?
+    - jak zorganizować całosć kodu?
+"""
+
+from PyQt5.QtWidgets import QWidget, QApplication, QGridLayout, QLabel, QMessageBox # QGridLayout, QLabel - do obsługi myszy
+from PyQt5.QtGui import QPainter, QPen, QPainterPath
+from PyQt5.QtCore import Qt
+import sys
+
+class Mapa(QWidget):
+    # czy będzie dobrym pomysłem zdefiniowanie zmiennej klasowej tutaj (przed funkcjami) zawierającej informację o wymiarach okna (w i h)?
```

Comment:
Jeśli chcemy mieć to zahardcode'owane, to w sumie rzeczywiście najlepiej żeby to było w klasie. Choć docelowo pewnie chcemy, żeby ta wartość mogła być dowolnie ustawiana w konstruktorze.